### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
Crafting materials update.

Updates to Key Items (?).

Having issues translating 토병, no clue what that is. Line 2732.

### CODE REVIEW

**Comment by emailboxu** [comment]
Apologies if I have overlapping changes with someone else, the computer I'm working on is super slow and can't handle more than ~3 tabs so I can't cross reference with other people's work. Just going straight through the file top to bottom.

**Comment by megaoka** [comment]
No worries, I just saw you had made the change already. Great work :+1:

File: ITEM_en.tsv [hunk 1]
```diff
@@ -2282,8 +2282,8 @@ ITEM_20150317_002402	Mallardu Meat
 ITEM_20150317_002403	A good source of protein. Obtained from Mallardus.
 ITEM_20150317_002404	Mallardu Feather
 ITEM_20150317_002405	Mallardu Beak
-ITEM_20150317_002406	Flower of Violet
-ITEM_20150317_002407	Crystal of Violet
+ITEM_20150317_002406	Violet Flower
```
Comment (megaoka) [inline ITEM_en.tsv:2285]: Violet Flower -> Violet's Flower

File: ITEM_en.tsv [hunk 2]
```diff
@@ -2282,8 +2282,8 @@ ITEM_20150317_002402	Mallardu Meat
...
+ITEM_20150317_002407	Violet Crystal
```
Comment (megaoka) [inline ITEM_en.tsv:2286]: Violet Crystal -> Violet's Crystal

**Comment by imcgames** [comment]
토병 is kind of soldier. :+1:

**Pull request merged**